### PR TITLE
Add support for h3-pg and re-enable plv8

### PR DESCRIPTION
**Pull request by kelvich**

### CODE REVIEW

File: Dockerfile.compute-node-v14 [hunk 1]
```diff
@@ -79,12 +79,47 @@ RUN wget https://github.com/plv8/plv8/archive/refs/tags/v3.1.4.tar.gz && \
...
     echo 'trusted = true' >> /usr/local/pgsql/share/extension/plv8.control
 
+#
+# Layer "h3-pg-build"
+# Build h3_pg
+#
+FROM build-deps AS h3-pg-build
+COPY --from=pg-build /usr/local/pgsql/ /usr/local/pgsql/
+RUN apt update && \
+    apt install -y wget
```
[the remark this answers is not in the document]
Comment (kelvich) [reply Dockerfile.compute-node-v14]: Thx, fixed. Also removed several unnecessary packages.

File: Dockerfile.compute-node-v14 [hunk 2]
```diff
@@ -59,7 +59,7 @@ RUN wget https://download.osgeo.org/postgis/source/postgis-3.3.0.tar.gz && \
 # Build plv8
 #
 FROM build-deps AS plv8-build
-COPY --from=postgis-build /usr/local/pgsql/ /usr/local/pgsql/
+COPY --from=pg-build /usr/local/pgsql/ /usr/local/pgsql/
```
[the remark this answers is not in the document]
Comment (kelvich) [reply Dockerfile.compute-node-v14:65]: > previously, we copied the PostgreSQL installation directory with all previously compiled extensions into each subsequent step for the next extension to install its data in.

With that change, all 3 extensions are built in parallel -- saves a bunch of time

File: Dockerfile.compute-node-v14 [hunk 3]
```diff
@@ -79,12 +79,47 @@ RUN wget https://github.com/plv8/plv8/archive/refs/tags/v3.1.4.tar.gz && \
...
+    cp -R /h3/usr / && \
+    rm -rf build
+
+RUN wget https://github.com/zachasme/h3-pg/archive/refs/tags/v4.0.1.tar.gz -O h3-pg.tgz && \
+    tar xvzf h3-pg.tgz && \
+    cd h3-pg-4.0.1 && \
+    export PATH="/usr/local/pgsql/bin:$PATH" && \
+    make -j $(getconf _NPROCESSORS_ONLN) && \
+    make -j $(getconf _NPROCESSORS_ONLN) install && \
+    echo 'trusted = true' >> /usr/local/pgsql/share/extension/h3.control
```
[the remark this answers is not in the document]
Comment (kelvich) [reply Dockerfile.compute-node-v14]: added CMAKE_BUILD_TYPE=Release to ensure

File: Dockerfile.compute-node-v14 [hunk 4]
```diff
@@ -79,12 +79,47 @@ RUN wget https://github.com/plv8/plv8/archive/refs/tags/v3.1.4.tar.gz && \
...
+# Build h3_pg
+#
+FROM build-deps AS h3-pg-build
+COPY --from=pg-build /usr/local/pgsql/ /usr/local/pgsql/
+RUN apt update && \
+    apt install -y wget
+
+# packaged cmake is too old
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.24.2/cmake-3.24.2-linux-x86_64.sh && \
+    sh cmake-3.24.2-linux-x86_64.sh --skip-license --prefix=/usr/
```
[the remark this answers is not in the document]
Comment (kelvich) [reply Dockerfile.compute-node-v14]: Yes, it works. Also moved testing repo setup to the `build-deps` stage as 2 of 3 extensions need testing repo now

**Review by ololobus — APPROVED** [review]
Didn't get through it thoroughly, but looks OK at the first sight

**Pull request merged**